Examples/Io/Root/src/RootPropagationStepsWriter.cpp

```diff
@@ -1,6 +1,6 @@
 // This file is part of the Acts project.
 //
-// Copyright (C) 2017 CERN for the benefit of the Acts project
```

Comment:
This file wasn't modified I think. Can you revert the copyright line change?

Reply:
I added at the previous overstepping-PR the lines 70, 117, 186-188 and overlooked to change the date.